### ISSUE TITLE
Many bars cause wrong bar color

### ISSUE THREAD

**Issue by argnist** [highcharts/highcharts#3776]
My setup for bar:

``` javascript
 plotOptions: {
                    bar: {
                        dataLabels: {
                            enabled: true,
                            style: {
                                fontSize: '12px',
                                color: '#000'
                            }
                        },
                        colorByPoint: true,
                        colors: [greenColor, greenLightColor]
                    }
                },
```

When amount of data is small I see green colors. But when it's big (over 1000) - colors turn to black.

I tried in console:

``` javascript
for (var i=0; i<50;i++)
chart.series[0].data[0].remove(false);
chart.redraw();
```

Then if length of data less than 1000 colors are still black but if you hover column it turns to green.

Found it is colorByPoint issue: http://jsfiddle.net/p4mr51qc/

**Comment by pawelfus** [comment]
Workaround: http://jsfiddle.net/p4mr51qc/1/ 
Disable:  `turboThreshold: 0`

**Comment by sebastianbochan** [comment]
Fixed since HC v5.0
